baseplate/thrift/utils.py

```diff
@@ -0,0 +1,38 @@
+"""Helpers for working with Thrift."""
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+from thrift.transport.TTransport import TMemoryBuffer
+
+
+def serialize_thrift(protocol_factory, thrift_object):
```

Comment:
Do we actually need these? Thrift (or at least FBThrift) already has these helper methods.

https://github.snooguts.net/reddit/reddit-service-mailroom/blob/70dc51b9222bc3af80019f1738e8499cae53fd0b/mailroom/rabbitmq/serializer.py#L4

Reply:
Apache Thrift does not which is why I'm providing them here.

Reply:
OH! i'm an idiot and just found it after grepping again. totally different name though. sweet. will update.